chrome/content/ui.js

```diff
@@ -213,6 +214,7 @@ var ui = {
       ui.addPrefMenus();
       ui.addBuddyContextMenu();
     }).catch(function(err) { throw err; });
+    coniks.init();
```

Comment:
With the above changes, `coniks.init()` will return a promise for the `loadFiles`.  We should include it in this chain,

```
  ...
  ui.addBuddyContextMenu();
  return coniks.init();
}).catch(function(err) { throw err; });
```